libs/libc/netdb/lib_gethostbyaddrr.c

```diff
@@ -61,8 +60,10 @@
 
 struct hostent_info_s
 {
+  int       hi_addrtypes[CONFIG_NETDB_MAX_IPADDR + 1];
+  int       hi_lengths[CONFIG_NETDB_MAX_IPADDR + 1];
```

Comment:
is "+ 1" necessary?
hi_addrlist needs +1 for terminating NULL. i guess these two don't.

Reply:
Yes, it isn't necessary, but it make the struct neat and orderly

Reply:
@yamt do you want to remove +1?

Reply:
if they are not necessary, yes

Reply:
Done.